test/e2e/helpers/utils.ts

```diff
@@ -0,0 +1,3 @@
+export function sleep(ms: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms))
+}
```

Comment:
maybe add this along other test utils? since it has it's own dedicater forlder "helpers" already

```
test/
├── e2e
│   └── homepage-redirect.spec.ts
├── helpers
│   ├── cloudagent.ts
│   ├── companyHouse.ts
│   ├── connection.ts
│   ├── db.ts
│   ├── fixtures.ts
│   ├── logger.ts
│   ├── routeHelper.ts
│   ├── smtp.ts
│   └── util.ts <<<< here maybe?
...
```

Reply:
I did not want to mix utils for integration and e2e tests

Reply:
is this function used?